File: Container Registry Key Identifiable/container_registry_key.js
const Docker = require('dockerode');

const docker = new Docker();
const registryUsername = "your_registry_username_here";
const registryPassword = "your_registry_password_here";
const registryUrl = "your_registry_url_here";

docker.checkAuth({
  username: registryUsername,
  password: registryPassword,
  serveraddress: registryUrl
}, function(err, data) {
  console.log(data);
});

File: Container Registry Key Identifiable/container_registry_key.py
import docker

client = docker.from_env()
registry_username = "your_registry_username_here"
registry_password = "your_registry_password_here"
registry_url = "your_registry_url_here"

client.login(
    username=registry_username, password=registry_password, registry=registry_url
)
